### ISSUE TITLE
[Windows Terminal>Setting>Profile>Appearance]: Name of the list items present in 'Variables font axes and font feature' groups are not define.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19695]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

Repro Steps:
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'Settings' menu item.
3.Settings page will open.
4.Now navigate to 'Profile' group and select 'default'.
5.Default page will open.
6.Now navigate to 'Appearance' section and expand it.
7.Now navigate to ''Font Feature and Variable Font Axes' groups and add some fonts.
4.Now turn on the narrator using 'CTRL+Win+Enter' keys.
4.Now navigate to the added list items.
5.Observe the issue here.

User Experience:
Screen reader users cannot identify the purpose of the list items if the name is not define, making it difficult to access the font features or variable axes list items independently. This can lead to confusion and errors.

WCAG Reference Link:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Attachements:
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQDDNGyS2wqZSqMcVVEQGWSsAUUu2j6cV4KaCWWSHAwG_Wc?e=Vxtquh

### Expected Behavior

Name should be define for the all the list items present in 'Font Feature and Variable Font Axes' groups.

### Actual Behavior

Name of the list items present in 'Variables font axes and font feature' groups are not define. 
Screen reader only announce the position as 1 of 2 and 2 of 2 on receiving focus.

**Comment by carlos-zamora** [comment]
The screen reader is reading "1 of 1" or something like that because the automation property name is not defined for the item. Shouldn't be too bad. Just a matter of setting `AutomationProperties.Name`.

<img width="1913" height="540" alt="Image" src="https://github.com/user-attachments/assets/dbd805ee-6472-47e0-b229-01c3d4763571" />

https://github.com/user-attachments/assets/d8d0a649-0423-46c8-bf28-26a8efadc9d8
